packages/ndla-ui/src/MyNdla/ResourceDash/ResourceElement.tsx

```diff
@@ -8,41 +8,350 @@
+const Halfwrapper = styled.div<{ layout: LayoutProps }>`
+  width: 100%;
+  ${(props) =>
+    props.layout === 'listLarger' &&
+    css`
+      margin-top: ${spacing.xxsmall};
+      height: 100%;
+      display: flex;
+      flex-direction: column;
+    `}
+  ${(props) =>
+    props.layout === 'block' &&
+    css`
+      background-color: white;
+    `}
+  ${mq.range({ until: breakpoints.tabletWide })} {
+    margin-top: 0;
+    margin-top: ${spacing.xxsmall};
+  }
+`;
+
+const Tophalf = styled.div<{ layout: LayoutProps }>`
+  display: flex;
+  flex-direction: row;
+  ${(props) =>
+    props.layout === 'block' &&
+    css`
+      flex-direction: column;
+    `}
+  ${mq.range({ until: breakpoints.tabletWide })} {
+    flex-direction: column;
+  }
+`;
+
+const Bottomhalf = styled.div<{ layout: LayoutProps }>`
+  margin-left: ${spacing.small};
+  margin-right: ${spacing.small};
+  overflow: hidden;
+  text-overflow: ellipsis;
+  transition-duration: 0.5s;
+`;
+
+const TagsList = styled.div<{ layout: LayoutProps }>`
+  height: 100%;
+  display: flex;
+  list-style: none;
+  margin-right: ${spacing.medium};
+  display: flex;
+  ${(props) =>
+    props.layout === 'listLarger' &&
+    css`
+      align-items: flex-start;
+    `}
+  ${(props) =>
+    props.layout === 'block' &&
+    css`
+      height: 50px;
+    `}
+
+  li {
+    display: flex;
+    margin: 0;
+    align-items: center;
+    padding-right: ${spacing.xxsmall};
+    padding-left: ${spacing.xxsmall};
+    color: ${colors.brand.grey};
+    font-size: ${fonts.sizes(14)};
+  }
+  ${mq.range({ until: breakpoints.tabletWide })} {
+    margin-right: ${spacing.small};
+  }
+`;
+
+const MoreIcon = styled(HorizontalMenu)<{ layout: LayoutProps }>`
+  height: 100%;
+  margin-right: ${spacing.small};
+  fill: ${colors.brand.tertiary};
+  transform: scale(1.5);
+  ${mq.range({ until: breakpoints.tabletWide })} {
+    margin-right: ${spacing.xsmall};
+    margin-top: ${spacing.xsmall};
+  }
+`;
+
+type ResourceImageProps = {
+  alt: string;
+  src: string;
+};
+interface ResourceElementProps {
+  link: string;
   title: string;
-  rightSide?: ReactElement;
-  leftSide?: ReactElement;
+  resourceImage: ResourceImageProps;
+  topics: string[];
+  tags?: string[];
   description?: string;
   children?: ReactElement;
-};
+  layout: LayoutProps;
+}
+type LayoutProps = 'list' | 'listLarger' | 'block';
 
-const ResourceElement = ({ type, title, description, leftSide, rightSide, children }: ResourceElementProps) => {
-  // const { t } = useTranslation();
+const ResourceElement = ({
+  link,
+  title,
+  tags,
+  children,
+  resourceImage,
+  topics,
+  layout,
+  description,
+}: ResourceElementProps) => {
+  const { t } = useTranslation();
   return (
-    <ResourceElementWrapper type={type}>
-      <div>{leftSide}</div>
-      <div>
-        <h1>{title}</h1>
-        <p>{description}</p>
-        {children}
-      </div>
-      <div>{rightSide}</div>
+    <ResourceElementWrapper to={link} layout={layout}>
```

Comment:
Jeg tror vi bør omstrukturere litt for å forbedre UU. Se gjerne på Læringsressurser komponenten. Hit area bør begrenses til kun tekst på ressurs / evt med bilde. Den bør trolig også ha en aria-label? Itillegg forenkler vi det med ... (more) lenken osv

Reply:
Vi skipper dette, har gjort en avsjekk med Phillipa så skal være ok